contracts/stellar-interchain-token-service/src/contract.rs

```diff
@@ -672,6 +697,34 @@ impl InterchainTokenService {
         deployed_address
     }
 
+    fn deploy_token_manager_contract(
+        env: &Env,
+        token_id: BytesN<32>,
+        token_address: Address,
+        token_manager_type: TokenManagerType,
+    ) -> Address {
+        let deployed_address = env
+            .deployer()
+            .with_address(
+                env.current_contract_address(),
+                Self::token_manager_salt(env, token_id.clone()),
+            )
+            .deploy_v2(
+                Self::token_manager_wasm_hash(env),
+                (env.current_contract_address(),),
+            );
+
+        TokenManagerDeployedEvent {
+            token_id,
+            token_manager_address: deployed_address.clone(),
+            token_address,
```

Comment:
`token_address` first, then `token_manager_address` to have the same order as `TokenIdConfigValue` 
```suggestion
            token_address,
            token_manager_address: deployed_address.clone(),
```

Reply:
surprised that cargo fmt doesn't auto-handle it